llvm/include/llvm/Support/AutoConvert.h

```diff
@@ -55,14 +55,14 @@ std::error_code restorezOSStdHandleAutoConversion(int FD);
 /** \brief Set the tag information for a file descriptor. */
 std::error_code setzOSFileTag(int FD, int CCSID, bool Text);
 
-// Get the the tag ccsid for a file name or a file descriptor.
+/** Get the the tag ccsid for a file name or a file descriptor. */
 ErrorOr<__ccsid_t> getzOSFileTag(const char *FileName, const int FD = -1);
 
-// Query the file tag to determine if it needs conversion to UTF-8 codepage.
+/** Query the file tag to determine if it needs conversion to UTF-8 codepage. */
 ErrorOr<bool> needzOSConversion(const char *FileName, const int FD = -1);
 
```

Comment:
While this is disabled in C code, I made these changes to unify style in whole file